content/blog/2024/pkg-importer.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Setting up Sass pkg URLs in Webpack
+sub: A quick guide to using the new Node package importer
+author: james
+date: 2024-02-21
+image:
+  src: blog/2024/sass-package.jpg
+  alt: >
+    Stacks of a variety of cardboard and food boxes,
+    some leaning precariously.
+  position: top
+tags:
+  - Article
+  - Sass
+  - Package management
+summary:
+  Enabling `pkg:` URLs is quick and straightforward
+  in projects using Vite or Webpack `sass-loader`,
```

Comment:
I'm curious about this line... It seems even _easier_ in projects using the Sass API directly -- if we want to call out Vite or Webpack here, I don't think we should make it sound like they make the process simpler.

Reply:
Good point, I'll remove that line.